src/include/storage/block_compactor.h

```diff
@@ -49,6 +54,36 @@ class BlockCompactor {
   };
 
  public:
+  BlockCompactor(execution::exec::ExecutionContext *exec, col_id_t *col_oids, const char *table_name) {
```

Comment:
Some documentation on this constructor can be helpful.

Reply:
Resolved.